package.json

```diff
@@ -13,7 +13,7 @@
     "start": "ts-node ./index.ts"
   },
   "name": "emojibase-bindings",
-  "version": "1.0.0",
+  "version": "0.0.1",
```

Comment:
We should avoid using 0.x as they're all treated unstable and don't conform to typical semver rules